Documentation/ceph-filesystem-crd.md

```diff
@@ -39,6 +39,7 @@ spec:
     - failureDomain: host
       replicated:
         size: 3
+  preservePools: false
```

Comment:
if I understand this correctly we don't  need to check is there any images/subdirectory present in pool or filesystem when `preservePoolsOnDelete` is set to false right

or do we need to provide a force option to delete the  pool/filesystem even if images or subdirectory present

Reply:
Assuming we add the pool option to force pool deletion if images exist, that option could be set on the pools. So I would expect something like this if you wanted to force delete:
```
  metadataPool:
    preserveIfImagesExistOnDelete: false
    failureDomain: host
    replicated:
      size: 3
  dataPools:
    - failureDomain: host
      preserveIfImagesExistOnDelete: false
      replicated:
        size: 3
  preservePools: false
```

Reply:
looks good, are we going to add it in this PR or will make it as separate PR?